src/DotLiquid/Liquid.cs

```diff
@@ -55,6 +55,7 @@ static Liquid()
             Template.RegisterTag<Tags.Html.TableRow>("tablerow");
 
             Template.RegisterFilter(typeof(StandardFilters));
+            Template.RegisterFilter(typeof(ShopifyFilters));

```

Comment:
As discussed on gitter, this should not be part of the library. MS will make a way to opt in outside of DLL.

Reply:
How about an alternative, which would enable a Designer to add a non-standard filter class to the render context.

For example: `{%-register_filters 'DotLiquid.ShopifyFilters'-%}`
The `register_filters` tag would add the DotLiquid.ShopifyFilters class using Context.AddFilters during rendering.